### PR TITLE
Update python Getting Started docs to include metrics

### PR DESCRIPTION
**Pull request by aabmass**
Tracking in https://github.com/open-telemetry/opentelemetry-python/issues/2732

This updates the Getting Started documentation for python with metrics instrumentation and the output of commands with metrics added.

### CODE REVIEW

File: content/en/docs/instrumentation/python/getting-started.md [hunk 1]
```diff
@@ -251,7 +260,147 @@ automatically created one:
...
+# These are the necessary import declarations
+from opentelemetry import trace
+from opentelemetry import metrics
+
+from random import randint
+from flask import Flask, request
+
+tracer = trace.get_tracer(__name__)
+# Acquire a meter. There's one set up for you globally,
+# and it's also used by opentelemetry-instrument.
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/getting-started.md:279]: Is this accurate? Did you mean to say provider is set up globally?

**Review by srikanthccv — APPROVED** [review]

File: content/en/docs/instrumentation/python/getting-started.md (hunk 1 above)
Comment (srikanthccv) [reply content/en/docs/instrumentation/python/getting-started.md:279]: Just remove the comment or keep `Acquire a {tracer,meter}.` part. I don't think there is any concept of setting up global tracer and use it. I believe the original author wanted to mention about the provider instead, which `opentelemetry-instrument` does in `initialize_components`.

Comment (aabmass) [reply content/en/docs/instrumentation/python/getting-started.md:279]: Good points, done.

**Comment by aabmass** [comment]
I'll be OOO for the next few weeks. Feel free to push any suggested changes to my branch and get this merged 🙂

**Review by ocelotl — APPROVED** [review]

**Comment by chalin** [comment]
@cartermp - any comments?

**Review by chalin — APPROVED** [review]
Thanks all!

**Comment by chalin** [comment]
@aabmass - could you rebase (resolve the conflict) and resubmit?

**Comment by cartermp** [comment]
All done!

**Review by chalin — APPROVED** [review]
Thanks all!

**Pull request merged**